modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -25,6 +25,18 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.securityAdmin`
 ====
 
+.Required permissions for creating service account resources
+[%collapsible]
+====
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+====
+
+[IMPORTANT]
+====
+The following permissions are required if you do not supply a service account for compute and control plane nodes in the `install-config.yaml` file. The permissions must be granted in the host project.
```

Comment:
- Do you mean "The above permissions are..." instead? 
- The permissions are required only if you do not supply a service account for **control plane** nodes in the `install-config.yaml` file. Please refer to [my testing](https://issues.redhat.com/browse/OSDOCS-11735?focusedId=25568016&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25568016), thanks!